elements/rh-footer/rh-footer-lightdom.css

```diff
@@ -141,6 +142,20 @@ rh-footer:not(:defined) {
   min-height: var(--rh-footer-nojs-min-height, 750px);
 }
 
+:is(rh-footer, rh-footer-universal):not(:defined) {
+  p {
+    color: var(--rh-color-text-secondary);
```

Comment:
You'll want to load the fallback value here for the color, unless the theming tokens are loaded directly on the page, which is unlikely. If the footer is the lone component and undefined, the theming tokens won't be available till after the JS loads.

```Suggestion
    color: 
      var(--rh-color-text-secondary, 
        light-dark(
            var(--rh-color-text-secondary-on-light, #4d4d4d),
            var(--rh-color-text-secondary-on-dark, #c7c7c7)
        ));
```